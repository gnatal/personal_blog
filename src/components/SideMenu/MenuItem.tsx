import React from 'react';

function MenuItem({
  linkURL,
  linkLabel,
}: {
  linkURL: string;
  linkLabel: string;
}) {
  return (
    <div className='w-screen pl-6 pb-2 pt-2 border'>
      <a
        href={linkURL}
        className='block font-bold text-md text-teal-100 hover:text-white'
      >
        {linkLabel}
      </a>
    </div>
  );
}

export default MenuItem;
