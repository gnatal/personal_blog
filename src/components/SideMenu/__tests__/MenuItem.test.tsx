import React from 'react';
import renderer from 'react-test-renderer';
import MenuItem from '../MenuItem';

test('It should show a link', () => {
  const component = renderer.create(<MenuItem linkLabel='home' linkURL='/' />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
