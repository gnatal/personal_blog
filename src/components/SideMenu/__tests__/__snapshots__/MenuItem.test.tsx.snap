// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should show a link 1`] = `
<div
  className="w-screen pl-6 pb-2 pt-2 border"
>
  <a
    className="block font-bold text-md text-teal-100 hover:text-white"
    href="/"
  >
    home
  </a>
</div>
`;
